include/RAJA/util/math.hpp

```diff
@@ -70,6 +70,37 @@ constexpr T next_pow2(T n) noexcept
   return n;
 }
 
+/*!
+    \brief "round down" to the next smallest power of 2
+
+    For a integer n,
+      if n is non-negative,
```

Comment:
I think this is clearer (since it looks more like the code):

if n is negative, return 0
else
   if n is a power of 2, return n
   else return the largest power of 2 that is less than n